dev-client/src/services.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+export const getElevation = async (lat: number, lng: number) => {
+  // TypeScript complains if the values passed to URLSearchParams are floats instead of strings.
+  // This API uses X for longitude and Y for latitude. That's not a typo.
+  const params = {
+    x: lng.toString(),
+    y: lat.toString(),
+    units: 'Meters', // TODO: switch based on user preference
+  };
```

Comment:
nit: it might be clearer to make the call to `new URLSearchParams` here. then any associated type errors will happen local to the definition, and it'd be more obvious what the params are for. obviously not a big deal for a function this small, but a good general style note i think